playground/mocks/data/idp/idx/device-trust-mtls-redirect.json

```diff
@@ -0,0 +1,48 @@
+{
+  "stateHandle": "inRUXNhsc6Evt7GAb8DPAA",
+  "version": "1.0.0",
+  "expiresAt": "2020-04-13T22:55:53.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "name": "redirect-device-trust",
+        "href": "http://dv1-devtrust.okta1.com/user/settings/certificate/challenge?relaystate=abcdUXb8DPNhsc6Evt7GXYa123",
```

Comment:
nit relaystate should be relayState + orgSubDomain is also needed.

Reply:
not really. We do not pass org subdomain here, thats a diff endpoint.